src/System.Drawing.Common/src/System.Drawing.Common.csproj

```diff
@@ -250,9 +250,13 @@
     </Compile>
   </ItemGroup>
   <ItemGroup>
-    <EmbeddedResource Include="Resources\System\Drawing\DefaultComponent.bmp" />
+    <EmbeddedResource Include="Resources\System\Drawing\DefaultComponent.bmp">
+      <LogicalName>System.Drawing.DefaultComponent.bmp</LogicalName>
+    </EmbeddedResource>
     <EmbeddedResource Include="Resources\System\Drawing\Printing\PrintDocument.bmp" />
```

Comment:
I can't find any uses of this, but am probably not looking hard enough. Maybe I should fix this resource as well, but I don't know how it's used